js/defaults_dialog.js

```diff
@@ -229,6 +229,30 @@ helper.defaultsDialog = (function() {
                     key: "small_angle",
                     value: 180
                 },
+                {
+                    key: "nav_fw_control_smoothness",
+                    value: 2
+                },
+                {
+                    key: "failsafe_procedure",
+                    value: "RTH"
```

Comment:
We should not do it as it requires optional sensors: GPS and Baro

Reply:
That is a fair point.